charts/fleet-telemetry/README.md

```diff
@@ -57,3 +60,99 @@ tlsSecret:
     value of the cert PEM
   tlsKey: |
     value the private key
+config:
+  data: |
+    {
+      "host": "0.0.0.0",
+      "port": 8443,
+      "status_port": 8080,
+      "log_level": "info",
+      "json_log_enable": true,
+      "namespace": "tesla_telemetry",
+      "reliable_ack": false,
+      "monitoring": {
+        "prometheus_metrics_port": 9273,
+        "profiler_port": 4269,
+        "profiling_path": "/tmp/trace.out"
+      },
+      "rate_limit": {
+        "enabled": true,
+        "message_interval_time": 30,
+        "message_limit": 1000
+      },
+      "records": {
+        "alerts": [
+          "logger"
+        ],
+        "errors": [
+          "logger"
+        ],
+        "V": [
+          "logger"
+        ]
+      },
+      "use_default_eng_ca": true,
```

Comment:
Please remove this key or set value to `false` as it should be rare to run the app in engineering mode.

Reply:
done